calcom/cal.com
# fix: request reschedule link in email for an Org event

## What does this PR do?
Fixes #11785

Even though this PR adds some tests for verifying links in email, the exact scenario of the bug is still not verified because request reschedule flow doesn't have tests at the moment. 

Created a ticket #12141 to test that.

## Type of change

## How should this be tested?
- Create an event in an org
- Do a booking for it
- Do 'Request Reschedule' for the event(from the app when logged in as the event organizer)
- Check the link in the email sent to booker 

## Mandatory Tasks

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
